work/worker.go

```diff
@@ -914,6 +910,31 @@ CommitTransactionLoop:
 		}
 	}
 
+	if atomic.LoadInt32(&abort) == 1 {
+		timeLimitReachedCounter.Inc(1)
+		var txHash common.Hash
+		if len(incorporatedTxs) > 0 {
+			txOrGen := incorporatedTxs[0]
+			tx, err := txOrGen.GetTx(env.state.GetNonce(nodeAddr))
+			if err == nil {
+				txHash = tx.Hash()
+			}
+			totalTxs := len(arrayTxs) + len(bundles)
```

Comment:
This isn't total txs. We need to check `BundleTxs` in each bundle.

Reply:
> Because bundle is all or nothing, I intended to indicate a single tx, rather than each tx in the bundle.

Then how do we use `env.tcount` for the `executed` field? Expanding the concept of bundle here breaks semantics for me. I think leaving `txs`, `bundleTxs` (`bundles` looks better to me) and deleting `totalTxs` would be better.

And I think we should also consider GA bundle here. The `approval`, `swap` tx will be included in `arrayTxs` since they're coming from the tx pool, not a separate channel like the auction bundle.